### PR TITLE
Fix XMLdoc for Vector<T> Inequality.

### PR DESCRIPTION
**Pull request by Gregory-Bell**
The != operator claims that it "Returns a value that indicates whether any single pair of elements in the specified vectors is equal." which is a copy-paste error from Vector.EqualsAny<T>().

### CODE REVIEW

File: src/System.Numerics.Vectors/src/System/Numerics/Vector.cs [hunk 1]
```diff
@@ -2752,11 +2752,11 @@ public string ToString(string format, IFormatProvider formatProvider)
         }
 
         /// <summary>
-        /// Returns a boolean indicating whether any single pair of elements in the given vectors are equal.
+        /// Returns a boolean indicating whether each pair of elements in the given vectors are not equal.
```
Comment (danmosemsft) [inline src/System.Numerics.Vectors/src/System/Numerics/Vector.cs]: Isn't it "any pair" ? It is not necessary for all ("each") to be not equal.

**Comment by danmosemsft** [comment]
CI failure is https://github.com/dotnet/corefx/issues/14775 and can be ignored.

File: src/System.Numerics.Vectors/src/System/Numerics/Vector.cs (hunk 1 above)
Comment (mellinoe) [inline src/System.Numerics.Vectors/src/System/Numerics/Vector.cs]: I guess it depends on how you interpret the wording here, but I also find the original wording clearer (although it's saying the exact opposite of what it means 😄).

Can we change it to this?
```
Returns a boolean indicating whether any single pair of elements in the given vectors are not equal.
```

(essentially just adding a "not" before "equal"). Let me know if that wording doesn't make sense for you, @Gregory-Bell

**Comment by mellinoe** [comment]
In order to change this file, you actually need to modify `Vector.tt` in the same folder, and then re-run the T4 generation tool from Visual Studio (just save the file in VS and it will run the T4 processor). Otherwise this change would just get wiped out whenever we modified `Vector.tt` again.

**Comment by Gregory-Bell** [comment]
@mellinoe that wording does sound better. I'll make that change in the T4 file when I get to my PC tonight.

File: src/System.Numerics.Vectors/src/System/Numerics/Vector.cs [hunk 2]
```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics.Contracts;
```
Comment (Gregory-Bell) [inline src/System.Numerics.Vectors/src/System/Numerics/Vector.cs]: This appeared all on its own when I ran the T4 compiler.

**Comment by danmosemsft** [comment]
Thanks @Gregory-Bell !

File: src/System.Numerics.Vectors/src/System/Numerics/Vector.cs (hunk 2 above)
Comment (mellinoe) [inline src/System.Numerics.Vectors/src/System/Numerics/Vector.cs]: This was caused by a previous change being applied only to the .cs file and not to the .tt file. It's not related to your stuff, but can you remove this line from the .tt file as well?

**Pull request merged**